src/main/scala/nl/knaw/dans/easy/stage/EasyStageDataset.scala

```diff
@@ -53,12 +56,13 @@ object EasyStageDataset {
     log.info("Creating dataset SDO")
     for {
       sdoDir <- mkdirSafe(new File(s.sdoSetDir, DATASET_SDO))
-      _ <- AMD.create(sdoDir)
+      _ <- writeAMD(sdoDir, AMD(s.ownerId, s.submissionTimestamp).toString())
```

Comment:
One of the commit messages explained. Without much prior knowledge I got very confused which file was used for what purpose. So I separated concerns in the AMD JSON and similar objects to only assemble content and not read/write anything. But indeed the toString could move into the AMD.apply method.